app/ui/omni-box.js

```diff
@@ -107,6 +107,19 @@ class OmniBox extends HTMLElement {
     this.forwardButton.addEventListener('click', () => {
       this.dispatchEvent(new CustomEvent('forward'))
     })
+
+    // middle mouse click paste&go
+    this.input.addEventListener('paste', (e) => {
+      e.preventDefault()
+      const url = e.clipboardData.getData('text')
+
+      if (!isURL(url)) {
```

Comment:
Could you invert this if so it'd be `if isURL { do the stuff }` instead of putting the main logic in the else.

Reply:
I'm on it